src/components/Button2.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./Button2.css";
+
+const Button2 = ({ text, btnClass, href }) => {
```

Comment:
More descriptive names could have been used (helpful to remember which button is which)